vars/buildAndProvideDebianPackage.groovy

```diff
@@ -64,7 +68,7 @@ def call(Boolean isArchIndependent = false, List ignoredArchs = []) {
             }
           }
           stage('Build binary - amd64') {
-            agent { label 'amd64' }
+            agent { label (isHeavyPackage ? 'amd64-fast' : 'amd64') }
```

Comment:
I think this is indeed a good idea, as we plan to move away from *in host* building at some point. this way it will be easier move (or recover) server

Reply:
I added amd64-fast to the current in-host node